portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,201 @@
+}
+
+/* Style the text selection color */
+ 
+::selection {
+    color: rgb(80, 80, 80); 
+    background: rgba(230, 175, 184, .822);
+}
+
+::-moz-selection { 
+    color: rgb(80, 80, 80);
+    background: rgba(230, 175, 184, .822);
+}
+
+/* Style different headings and paragraph text */
+
+h1 { 
+  font-family: 'Poppins';
+  font-size: 90px; 
+  font-style: bold;
+  font-weight: 700; 
+  line-height: 26.4px; 
+} 
+
+h2 {
+  font-family: 'Poppins'; 
+  font-size: 24px;  
+  font-variant: normal;  
+  font-weight: 400;
+  line-height: 23px; 
+}
+
+h3 { 
+  font-family: 'Montserrat'; 
+  font-size: 24px; 
+  font-style: normal; 
+  font-variant: small-caps; 
+  font-weight: 400;  
+  margin: 0px;
+}
+
+p {
+  font-family: 'Montserrat'; 
+  font-size: 15px; 
+  font-style: normal; 
+  font-variant: normal; 
+  font-weight: 400; 
+}
+
+
+/* Style the header section */
+
+.header { 
+  background-color: rgb(255, 177, 104);
+  box-shadow: 0px 5px 39px 2px rgba(69,69,69,0.71);
+  -moz-box-shadow: 0px 5px 39px 2px rgba(69,69,69,0.71);
+  -webkit-box-shadow: 0px 5px 39px 2px rgba(69,69,69,0.71);
+  color: white;
+  grid-area: header;
+  overflow: hidden;
+  position: fixed; 
+  top: 0; 
+  width: 100%; 
+}
+
+nav {
+    margin-left: 150px;
+    margin-top: 20px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* Style the links inside the navigation bar */
+
+nav a {
+  color: white; 
+  float: left; 
+  font-family: "Poppins"; 
+  font-size: 17px;
+  font-weight: 300; 
+  padding: 0 20px 20px 20px;
+  text-align: center;
+  text-decoration: none;
+}
+
+nav a:hover {
+    color: rgb(255, 221, 219);
+}
+
+nav a.active {
+    background-color: rgb(235, 200, 186);
+    color: white;
+}
+
+/* Style the introduction section */
+
+.intro { 
+  background: url(/images/tybee.jpg) no-repeat center center fixed;
+  background-size: cover;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  color: white;
+  grid-area: intro; 
+  padding: 200px 150px 0 150px;
+  text-align: left;
+}
+.intro h2 {
+  font-style: italic;
+}
+.intro h3 {
+  font-size: 30px;
+}
+
+/* Style the about section */
+
+.about {
+  color: hsl(240, 3%, 14%);
+  background-color: rgb(255, 245, 242);
+  padding: 30px 150px 50px 150px;
+  grid-area: about;
+  vertical-align: middle;
+}
+
+.about img {
+  float: left;
+  height: 400px;
+  margin-right: 50px;
+  width: 300px;
+}
+
+/* Style the footer/contact section */
+
+.footer {
+  background-color: rgb(255, 177, 104);
+  color: white;
+  padding-left: 150px;
+  padding-right: 150px; 
+  padding-top: 15px;
+  grid-area: footer; 
+}
+
+.footer a {
+  color: white;
+  text-decoration: none;
+  font-size: 17px;
+  font-family: 'Poppins'; 
+  font-style: italic; 
+  font-weight: 300; 
+}
+
+.footer a:hover {
+  color: rgb(255, 221, 219);
+}
+
+.footer .social a {
+  color: white;
+  text-align: center;
+  text-decoration: none;
+  font-size: 25px; 
+  font-style: normal;
+  padding-bottom: 15px;
+  padding: 3px;
```

Comment:
When applying the same style in one class, CSS will override the top most styles with the last style applied. So here with both paddings, it only applies the bottom one i.e. 
`padding: 3px;`
Unsure if you mean to only use padding 3px or also have 15px for the bottom padding

Reply:
Oops, I meant for top, left, and right to have 3px and bottom to have 15px. So I corrected this to `padding: 3px 3px 15px`